spec/features/user_edits_beach_spec.rb

```diff
@@ -107,7 +107,7 @@
       expect(page).to_not have_content('Delete Beach')
     end
 
-    scenario "malicious user attempts to edit someone else's review" do
+    scenario "malicious user attempts to edit someone else's beach" do
```

Comment:
Small detail, but the user doesn't have to be malicious to attempt to change another's beach. You're just preventing a non-admin, non-beach-user from doing this, so it might be worth updating this accordingly.